### ISSUE TITLE
Method MandateClearedFromBillable::handle() does not exist

### ISSUE THREAD

**Issue by meesudzu** [laravel/cashier-mollie#109]
I got an error when call the webhook from mollie.
I think this error from the new version.

```
ReflectionException: Method Laravel\Cashier\Events\MandateClearedFromBillable::handle() does not exist in file /home/mollie/public_html/vendor/laravel/framework/src/Illuminate/Container/BoundMethod.php on line 144
```
Update: I back to v1.1.0 and everything working ok.

**Comment by sandervanhooft** [comment]
Thanks! Found it. Working on a patch.

**Comment by sandervanhooft** [comment]
Hi @meesudzu ,

I've just released `v1.6.0` which includes a patch to fix this.

**Comment by meesudzu** [comment]
> Hi @meesudzu ,
> 
> I've just released `v1.6.0` which includes a patch to fix this.

thanks for hard working.
